### ISSUE TITLE
Database error for MySQL InnoDB database without innodb_large_prefix extension

### ISSUE THREAD

**Issue by Zodiac1978** [pluginkollektiv/statify#105]
First reported here:
https://wordpress.org/support/topic/fehler-bei-der-aktivierung-des-plugins-tabelle-wp_statify-wird-nicht-angelegt/

Could we check for this extension, catch this error and act accordingly?
Contao seems to be doing this:
https://github.com/contao/installation-bundle/commit/1bdf0ec2c8bc9472ee9b3cdb8c6e0f2dd0696659

**Comment by stklcode** [comment]
Sure we can check for availability of the extension, we just jave to agree on a proper resoltion.
1. Properly fail installation with a readable warning
2. Simply skip the index creation and live with the performance impact 
3. Find a different solution like hashing the content and building an index table on that fixed-length hash?

Performance shouldn't be that much of an issue and only affects admin dashboard on sites with a high number of users, so I think version 2 is the way to go.

**Comment by stklcode** [comment]
I adapted the Contao checks, but I think their implementation is incomplete.

I tested on serveral servers and at least on one (Docker `mariadb:10.1.36-bionic`) the check returns `false`, but Index creation is possible.

```
Variable_name             | Value           || Condition
--------------------------|-----------------||-----------------------
innodb_large_prefix       | OFF             || not ON => continue
version                   | 10.1.36-MariaDB || < 10.2 => continue
innodb_file_format        | Antelope        || not Barracuda => fail
innodb_default_row_format | compact         || 
innodb_file_per_table     | ON              || 
```

To everything I read is should not be possible... Didn't check if this is a somehow patched 10.1 release.

**Comment by bueltge** [comment]
The issue is ancient and not validated in several installations. So I mean we should close the issue for now and if we get it again, then restart the analysis.
